include/isaac/value_scalar.h

```diff
@@ -34,7 +34,7 @@ namespace isaac
 class scalar;
 class expression_tree;
 
-union ISAACAPI values_holder
+struct ISAACAPI values_holder
 {
```

Comment:
This is because of the introducing of class half, use ```struct``` can avoid compiling error.